src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -1178,7 +1192,10 @@ class IpexCommunicationService extends AgentService {
       };
     }
 
-    return result;
+    return {
+      threshold: multisigAid.state.kt,
+      ...result,
```

Comment:
Hmm, I think it should be more like `{ threshold: number, offers: result }` or something. Otherwise threshold will be beside credential IDs and that's confusing